src/connect-actions.ts

```diff
@@ -90,13 +91,15 @@ export class ConnectActions {
   };
 
   private flashMicrobit = async (
-    flowType: ConnectionFlowType,
+    hex: string,
     progress: (progress: number) => void
   ): Promise<ConnectAndFlashResult> => {
     if (!this.usb) {
       return ConnectAndFlashResult.Failed;
     }
-    const data = getFlashDataSource(flowType);
+    const data = Object.values(HexType).includes(hex as HexType)
```

Comment:
I tried this, but when doing `hex` is "radio-remote", `(hex in HexType)` is resolved as `false` instead of `true`. Turns out this does not work with string enums because they are not reverse mapped [see this forum](https://stackoverflow.com/questions/43804805/check-if-value-exists-in-enum-in-typescript/47755096#47755096)

Reply:
Well that's disappointing. Nevermind.